src/components/containers/ChartContainer.js

```diff
@@ -34,6 +38,7 @@ export default class extends React.Component {
 
         data.populations.forEach((population, index) => {
             const thisColor = this.colors[index];
+            let resultData = [];
```

Comment:
Prevents us from modifying dataPoint.y permanently. This makes me think moving the format call outside of render should be possible again since that was my original issue...will look into it.